assets/xml/objects/object_po.xml

```diff
@@ -1,53 +1,65 @@
-        <Limb Name="object_po_Standardlimb_005038" Type="Standard" EnumName="OBJECT_PO_LIMB_0F" Offset="0x5038" />
-        <Limb Name="object_po_Standardlimb_005044" Type="Standard" EnumName="OBJECT_PO_LIMB_10" Offset="0x5044" />
-        <Limb Name="object_po_Standardlimb_005050" Type="Standard" EnumName="OBJECT_PO_LIMB_11" Offset="0x5050" />
-        <Limb Name="object_po_Standardlimb_00505C" Type="Standard" EnumName="OBJECT_PO_LIMB_12" Offset="0x505C" />
-        <Limb Name="object_po_Standardlimb_005068" Type="Standard" EnumName="OBJECT_PO_LIMB_13" Offset="0x5068" />
-        <Limb Name="object_po_Standardlimb_005074" Type="Standard" EnumName="OBJECT_PO_LIMB_14" Offset="0x5074" />
-        <Skeleton Name="object_po_Skel_0050D0" Type="Normal" LimbType="Standard" LimbNone="OBJECT_PO_LIMB_NONE" LimbMax="OBJECT_PO_LIMB_MAX" EnumName="ObjectPoLimb" Offset="0x50D0" />
+        <!-- Animations -->
+        <Animation Name="gPoeAttackAnim" Offset="0x1A8" />
+        <Animation Name="gPoeDamagedAnim" Offset="0x4EC" />
+        <Animation Name="gPoeFleeAnim" Offset="0x6E0" />
+        <Animation Name="gPoeFloatAnim" Offset="0xA60" />
+        <Animation Name="gPoeAppearAnim" Offset="0x11C4" />
+        <Animation Name="gPoeDisappearAnim" Offset="0x15B0" />
+
+        <!-- Poe Limb DisplayLists -->
+        <DList Name="gPoeRightUpperArmDL" Offset="0x1620" />
+        <DList Name="gPoeLeftForearmDL" Offset="0x1898" />
+        <DList Name="gPoeLeftUpperArmDL" Offset="0x1A78" />
+        <DList Name="gPoeRightForearmDL" Offset="0x1B70" />
+        <DList Name="gPoeRightHandDL" Offset="0x1C68" />
+        <DList Name="gPoeCloak1DL" Offset="0x1FB0" />
+        <DList Name="gPoeBurnDL" Offset="0x2608" />
+        <DList Name="gPoeFaceDL" Offset="0x2910" />
+        <DList Name="gPoeLanternDL" Offset="0x2D28" />
+        <DList Name="gPoeCloak2DL" Offset="0x2F58" />
+
+        <!-- Poe Soul -->
+        <Texture Name="gPoeSoulTex" OutName="poe_soul" Format="i8" Width="32" Height="64" Offset="0x3010" />
+        <DList Name="gPoeSoulDL" Offset="0x3850" />
+
+        <!-- Textures -->
+        <Texture Name="gPoeArmTex" OutName="poe_arm" Format="rgba16" Width="32" Height="16" Offset="0x3910" />
+        <Texture Name="gPoeCloak1Tex" OutName="poe_cloak1" Format="rgba16" Width="32" Height="32" Offset="0x3D10" />
+        <Texture Name="gPoeCloak2Tex" OutName="poe_cloak2" Format="rgba16" Width="16" Height="16" Offset="0x4510" />
+        <Texture Name="gPoeCloak3Tex" OutName="poe_cloak3" Format="rgba16" Width="8" Height="8" Offset="0x4710" />
+        <Texture Name="gPoeEyeTex" OutName="poe_eye" Format="rgba16" Width="16" Height="16" Offset="0x4790" />
+        <Texture Name="gPoeLanternTopTex" OutName="poe_latern_top" Format="rgba16" Width="8" Height="8" Offset="0x4990" />
+        <Texture Name="gPoeLanternBottemTex" OutName="poe_latern_bottem" Format="rgba16" Width="8" Height="16" Offset="0x4A10" />
```

Comment:
```suggestion
        <Texture Name="gPoeLanternBottomTex" OutName="poe_lantern_bottom" Format="rgba16" Width="8" Height="16" Offset="0x4A10" />
```
Presumably? Applies to a few places in this file (both latern typo and bottem typo)